content/release-10-14-0/streaming-analytics-10-14-0-bundle/10_14_0_3.md

```diff
@@ -0,0 +1,34 @@
+---
+weight: 37
+title: Release 10.14.0.3
+layout: redirect
+---
+
+In this release, the Apama-ctrl microservice uses the same Apama version as in the previous 10.14.0.2 release. 
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Streaming Analytics application</td>
+<td style="text-align:left">The Streaming Analytics application has been updated to use the 10.14.0.133 Web SDK. 
+  See also <a href="/release-10-14-0/applications-10-14-0/">Applications<a/>.</td>
```

Comment:
I _think_ this URL is wrong. When 'live' , it'll take you to https://cumulocity.com/release-10-14-0/applications-10-14-0/ rather than https://cumulocity.com/releasenotes/release-10-14-0/applications-10-14-0/ . Might need to be a relative URL e.g "../applications-10-14-0/".

Reply:
Hi @BeateRixen , what is your opinion about the URL? I tried it out in Hugo and it worked.

Reply:
> 

@kailol-sag This is the way we always do it and it works :-).